ledfx/utils.py

```diff
@@ -58,7 +82,12 @@ def fps_to_sleep_interval(fps):
         (t for f, t in AVAILABLE_FPS.items() if f >= fps),
         list(AVAILABLE_FPS.values())[-1],
     )
-    return max(0.001, monotonic_res * (monotonic_ticks - 1))
+    if sleep_lies:
+        min_clock = monotonic_res
+    else:
+        min_clock = 0.001
+    _LOGGER.warning(f"MIN clock is {min_clock}")
+    return max(min_clock, monotonic_res * (monotonic_ticks - 1))
```

Comment:
Modify the minimum timing according to the scenario through min_clock. This is the core of the fix